tests/lib/managers/retention-policies-test.js

```diff
@@ -326,6 +326,31 @@ describe('RetentionPolicies', function() {
 			retentionPolicies.assign(POLICY_ID, ASSIGN_TYPE, ASSIGN_ID);
 		});
 
+		it('should make POST request to assign policy with optional params when options are passed', function() {
+
+			var options = {
+				filter_fields: [
+					{
+						field: 'foo',
+						value: 'bar'
+					},
+					{
+						field: 'baz',
```

Comment:
Looks good! Can you specify more than one filter fields currently though? From the Note in the md